lib/wikicloth/core_ext.rb

```diff
@@ -34,7 +36,13 @@ def auto_link
     end
   else
     def auto_link
-      Twitter::Autolink.auto_link(to_s)
+      doc = Nokogiri::HTML::DocumentFragment.parse(to_s)
```

Comment:
We might consider doing:
```ruby
next if node.parent.name == 'a'
```
...so that URL's within `<a>` tags won't be autolinked.